pkg/controller/podgc/gc_controller.go

```diff
@@ -38,6 +38,8 @@ import (
 
 const (
 	gcCheckPeriod = 20 * time.Second
+	purgePeriod   = 20 * time.Minute
```

Comment:
Can we add a documentation for these constants?

Reply:
Done